index.html

```diff
@@ -51,5 +51,154 @@ <h1>
         <li> <img src="./images/angellist.png" alt=""></li>
       </ul>
       </section>
+  
+      <section class="card-Work-container">
+        <div class="snapshot-porfolio">
+          <img src="./images/snap 1.png" alt="">
+        </div>
+        <h2>Tonic</h2>
+        <ul class="tonic">
+          <li class="canopy">CANOPY</li>
+          <li class="back-end">Back End Dev</li>
+          <li class="year">2015</li>
+        </ul>
+        <p>A daily selection of privately 
+          personalized reads; no accounts 
+          or sign-ups required.</p>
+          <ul class="tag">
+            <li class="html">html</li>
+            <li class="cascade">css</li>
+            <li class="javascript">javascript</li>
+          </ul>
+
+          <div class="see-project-one">
+           <button class="see-project">See project</button>
+           </div>
+
+      </section>
+
+      <section class="card-Work-container-2">
+        <div class="snapshot-porfolio">
+          <img src="./images/snap 2.png" alt="">
+        </div>
+        <h2>Multi-Post <br> Stories</h2>
+        <ul class="tonic">
+          <li class="canopy">CANOPY</li>
+          <li class="back-end">Back End Dev</li>
+          <li class="year">2015</li>
+        </ul>
+        <p>A daily selection of privately 
+          personalized reads; no accounts 
+          or sign-ups required.</p>
+          <ul class="tag">
+            <li class="html">html</li>
+            <li class="cascade">css</li>
+            <li class="javascript">javascript</li>
+          </ul>
+          <div class="see-project-one">
+           <button class="see-project">See project</button>
+           </div>
+
+      </section>
+
+      <section class="card-Work-container">
+        <div class="snapshot-porfolio">
+          <img src="./images/snap 3.png" alt="">
+        </div>
+        <h2>Tonic</h2>
+        <ul class="tonic">
+          <li class="canopy">CANOPY</li>
+          <li class="back-end">Back End Dev</li>
+          <li class="year">2015</li>
+        </ul>
+      
+        <div>
+        <p>A daily selection of privately 
+          personalized reads; no accounts 
+          or sign-ups required.</p>
+          </div>
+          <ul class="tag">
+            <li class="html">html</li>
+            <li class="cascade">css</li>
+            <li class="javascript">javascript</li>
+          </ul>
+
```

Comment:
- [ ] Please use display grid to align your work section, as mentioned in the project requirements _**In order to lay out the elements in "Works" section you should use [CSS Grid ](https://css-tricks.com/snippets/css/complete-guide-grid/)and [Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.**_
- [ ]  I think it would be best to group all of your work `section` in a single section tag, and use `div` for each individual project card, as this will give our site a better structure and will allow us more freedom with styling the whole section 
example: 
```
<section class="Works">
  <div class="card-one>
   project card content
  </div>
 <div class="card-two> 
   project card content
  </div>
 <div class="card-three>
  project card content
  </div>   
<div class="card-four>
project card content
  </div>
</section>
```
Then you can use `display: grid `on the works section and flexbox for each individual project cards :+1: